contracts/mocks/TellorLibraryTest.sol

```diff
@@ -41,135 +41,135 @@ library TellorLibraryTest {
     }
     //Outdated Functions needed for upgrading
```

Comment:
Is this one still needed or can remove it completely instead of commenting out?

Reply:
We can remove... but it wasn't supposed to be in the PR. I was changing a bunch of stuff across different branches and that must been included here by accident